### DISCUSSION TITLE
Old Plate Texture Pack

### DISCUSSION THREAD

**Discussion by Trent2Luigi** [terrarium-earth/Ad-Astra#242] [Q&A] [answered]
I've made a build with the plated pillars and blocks, I liked how it matched the Copper Blocks Textures, but now they look terrible in my build. Is it possible to get the old textures back at a Texture pack? And/or willing to do so/ have already done so for me? No offence to the new textures, but it doesn't fit my pallete anymore, and I want to update my game without ruining my build.

**Comment by CodexAdrian** [comment, marked as answer]
We won't be doing that. Our artists spend a ton of time working on the art they create, and these new pillars were made with alot of attention to detail. Adjust to the new textures of the blocks, reverting the work these amazing artists have done is not something we will entertain.

**Reply by Trent2Luigi** [reply]
Sorry, I didn't mean it like that. I was just wanted an option for an alternate look, Not to undo anybody's work. Like the Title Says, I just wanted a Texture pack, not a full change for everyone, Just an option for those how wanted it. I mean, people don't like change sometimes, was it wrong to think someone would have done something like that to keep that texture?

I really didn't mean to step on any toes, I really do support this mod 100%, I love it and really am excited to see where it goes. I mean slightly like I was made out to be rude in your reply, but maybe the message sounded better in my head, I'm not great with words sometimes. Everyone is doing great on this mod, I love the way it feels like it fits into the new vanilla style... But i felt like the old plates fit that style more, and I'll Probably have to get used to it.

Sorry again, and I hope everyone keeps up the good work.

**Reply by CodexAdrian** [reply]
> Sorry, I didn't mean it like that. I was just wanted an option for an alternate look, Not to undo anybody's work. Like the Title Says, I just wanted a Texture pack, not a full change for everyone, Just an option for those how wanted it. I mean, people don't like change sometimes, was it wrong to think someone would have done something like that to keep that texture?
> 
> 
> 
> I really didn't mean to step on any toes, I really do support this mod 100%, I love it and really am excited to see where it goes. I mean slightly like I was made out to be rude in your reply, but maybe the message sounded better in my head, I'm not great with words sometimes. Everyone is doing great on this mod, I love the way it feels like it fits into the new vanilla style... But i felt like the old plates fit that style more, and I'll Probably have to get used to it.
> 
> 
> 
> Sorry again, and I hope everyone keeps up the good work.

We as a studio have to consider a lot of things when we create content. It's not productive for us as a studio when we improve our existing content, which is what this is from every possible angle- a lot of thought and care was put into making these new assets better than the old ones, to make the old stuff we deemed was bad enough to need to be replaced to still exist. We cannot make everyone happy, but I don't think it's reasonable for us to expect that every time we make a change for improvement to make sure we keep around legacy stuff, not only because it's hard to maintain, but because the entire point of replacing the content was because it was bad. So no matter what your intent was with this, this isn't something we're going to do for Ad Astra or any mod for that matter, the changes we make are with intention and we do it to make our mods the best they can be.
